engines/qemu/config-schema.yml

```diff
@@ -0,0 +1,25 @@
+$schema:            http://json-schema.org/draft-04/schema#
+title:              "Engine Config"
+type:               object
+properties:
+  maxConcurrency:
+    type:           integer
+    minimum:        1
+    maximum:        100
+    description: |
```

Comment:
Who cares about trailing lines in markdown... (description is rendered as markdown in our docs)...

But good point...